oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/model/StorageModels.java

```diff
@@ -52,14 +55,17 @@ public Model add(Class<?> aClass, int scopeId, Storage storage, boolean record)
         // Check this scope id is valid.
         DefaultScopeDefine.nameOf(scopeId);
 
+        // extract properties from class
```

Comment:
What does this mean? This doesn't match the codes.

Reply:
already remove it